File: C/1012.c
#include <stdio.h>
#include <math.h>

int main() {
    double a, b, c;
     
    scanf("%lf %lf %lf", &a, &b, &c);

    printf("TRIANGULO: %.3f\n", (a * c)/2);
    printf("CIRCULO: %.3f\n", 3.14159 * c * c);
    printf("TRAPEZIO: %.3f\n", ((a + b) * c)/2);
    printf("QUADRADO: %.3f\n", b * b);
    printf("RETANGULO: %.3f\n", a * b);

    return 0;
}

File: 1. Beginner/1012/1012.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);

        double a = scanner.nextDouble();
        double b = scanner.nextDouble();
        double c = scanner.nextDouble();
        
        System.out.println(String.format("TRIANGULO: %.3f", a * c / 2.0));
        System.out.println(String.format("CIRCULO: %.3f", 3.14159 * (c * c)));
        System.out.println(String.format("TRAPEZIO: %.3f", ((a + b) * c) / 2));
        System.out.println(String.format("QUADRADO: %.3f",  b * b));
        System.out.println(String.format("RETANGULO: %.3f", a * b));  
    }
}